x-pack/plugins/security_solution/public/flyout/left/components/test_ids.ts

```diff
@@ -5,14 +5,20 @@
  * 2.0.
  */
 
-export const ANALYZER_GRAPH_TEST_ID = 'securitySolutionDocumentDetailsFlyoutAnalyzerGraph';
-export const ANALYZE_GRAPH_ERROR_TEST_ID =
-  'securitySolutionDocumentDetailsFlyoutAnalyzerGraphError';
-export const SESSION_VIEW_TEST_ID = 'securitySolutionDocumentDetailsFlyoutSessionView';
-export const SESSION_VIEW_ERROR_TEST_ID = 'securitySolutionDocumentDetailsFlyoutSessionViewError';
-export const ENTITIES_DETAILS_TEST_ID = 'securitySolutionDocumentDetailsFlyoutEntitiesDetails';
+const PREFIX = 'securitySolution' as const;
```

Comment:
I like this, but I think we could add another one `securitySolution` and not `securitySolutionDocumentDetailsFlyout` as I think we use this longer text everywhere?

Reply:
sounds reasonable